src/Context/notes-context.js

```diff
@@ -0,0 +1,68 @@
+import { useReducer, createContext, useContext } from "react";
+
+const NoteContext = createContext();
+
+const useNote = () => useContext(NoteContext);
+
+const noteReducer = (noteState, action) => {
+  console.log(action);
+  //   sort out unique label
```

Comment:
this is correct . just a suggestion you can destructure action into payload and type beforehand .